mopidy/config/__init__.py

```diff
@@ -182,6 +183,12 @@ def _validate(raw_config, schemas):
             errors[schema.name] = error
         if result:
             config[schema.name] = result
+        if schema.name in sections:
+            sections.remove(schema.name)
+
+    for section in sections:
+        logger.debug('Ignoring unknown config section: %s', section)
```

Comment:
Leaving it at debug level also makes it more usable for e.g. Pi Musicbox to use Mopidy's config as a general config store.

I'm OK with this as is.